Source/MQTTnet.AspnetCore/MqttBufferWriterPoolOptions.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace MQTTnet.AspNetCore
+{
+    public sealed class MqttBufferWriterPoolOptions
+    {
+        public bool Enable { get; set; } = true;
+
+        /// <summary>
+        /// When the life time of the MqttBufferWriter is less than this value, MqttBufferWriter is pooled.
+        /// </summary>
+        public TimeSpan PoolingItemMaxLifeTime { get; set; } = TimeSpan.FromMinutes(1d);
+
```

Comment:
Help, I really don't know how to name these properties here.

Reply:
As these are already in the pool options class, I suggest `MaxLifetime` and `LargeBufferSizeEnabled`